### PR TITLE
Fix BI Metrics app ver and os ver incorrect

### PR DESCRIPTION
**Pull request by maggie98choy**
## What

Set BI metrics app_ver field to ripple version and BI metrics os_ver field to firmware version

## Why

There are not being set correctly

## How

`app_ver` field is set to ripple version. 
`os_ver` field is set to the value from getSystemVersions thunder call.

## Checklist

- [x] I have self-reviewed this PR
- [ ] I have added tests that prove the feature works or the fix is effective

### CODE REVIEW

File: core/sdk/Cargo.toml [hunk 1]
```diff
@@ -20,6 +20,7 @@ name = "ripple_sdk"
 version = "1.0.0"
 edition = "2021"
 repository = "https://github.com/rdkcentral/Ripple"
+build = "build.rs"
```
Comment (satlead) [inline core/sdk/Cargo.toml:23]: Can we take this from main instead of SDK

File: core/sdk/build.rs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+extern crate vergen;
+use vergen::{vergen, ConstantsFlags};
+
+fn main() {
```
Comment (satlead) [inline core/sdk/build.rs:21]: SDK should not need a main

File: core/sdk/src/api/firebolt/fb_metrics.rs [hunk 3]
```diff
@@ -47,7 +48,7 @@ impl From<CallContext> for BehavioralMetricContext {
     fn from(call_context: CallContext) -> Self {
         BehavioralMetricContext {
             app_id: call_context.app_id.clone(),
-            app_version: String::from("app.version.not.implemented"),
+            app_version: SEMVER.to_string(),
```
Comment (satlead) [inline core/sdk/src/api/firebolt/fb_metrics.rs:51]: Main can replace this on the metrics context

File: core/sdk/build.rs (hunk 2 above)
Comment (brendanobra) [reply core/sdk/build.rs]: This main for build, not main (main) (build.rs, not main.rs). This is required to get the version to be generated correctly.

**Review by brendanobra — APPROVED** [review]

**Review by satlead — APPROVED** [review]

**Review by B-Larsen — APPROVED** [review]

**Pull request merged**